src/napari/_app_model/actions/_layerlist_context_actions.py

```diff
@@ -222,6 +231,34 @@
             }
         ],
     ),
+    Action(
+        id='napari.layer.bounding_box',
+        title=trans._('Bounding Box'),
+        callback=_layer_actions._toogle_bounding_box,
```

Comment:
```suggestion
        callback=_layer_actions._toggle_bounding_box,
```
(method in _layer_actions needs updated as well)